go/chat/helper.go

```diff
@@ -268,10 +268,11 @@ func GetTLFConversations(ctx context.Context, g *globals.Context, debugger utils
 	membersType chat1.ConversationMembersType) (res []chat1.ConversationLocal, rl []chat1.RateLimit, err error) {
 
 	tlfRes, err := ri().GetTLFConversations(ctx, chat1.GetTLFConversationsArg{
-		TlfID:            tlfID,
-		TopicType:        topicType,
-		MembersType:      membersType,
-		SummarizeMaxMsgs: false,
+		TlfID:                tlfID,
+		TopicType:            topicType,
+		MembersType:          membersType,
+		SummarizeMaxMsgs:     false,
+		IncludeAuxiliaryInfo: true,
```

Comment:
See my other comment.  Perhaps this should be controlled by argument to this function.